src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -506,6 +506,7 @@ internal void SelectAll(object parameter)
         {
             DynamoSelection.Instance.ClearSelection();
             Nodes.ToList().ForEach((ele) => DynamoSelection.Instance.Selection.Add(ele.NodeModel));
+            Notes.ToList().ForEach((ele) => DynamoSelection.Instance.Selection.Add(ele.Model));
```

Comment:
'Ele' is part of the foreach loop definition - in this case, iterating through the NoteViewModels of all the Notes held in the current workspaceviewmodel and adding the corresponding NoteModels (ele.Model) to the current dynamo selection